src/test/test_verify.hpp

```
@@ -113,8 +110,57 @@ inline std::vector<prevail::RawProgram> read_elf_cached(const std::string& path,
+    static constexpr size_t MAX_LEN = 75;
+    char data[MAX_LEN + 1]{};
+
+    template <size_t N>
+    explicit constexpr BoundedTestName(const char (&str)[N]) {
+        if constexpr (N - 1 <= MAX_LEN) {
+            for (size_t i = 0; i < N; ++i) {
+                data[i] = str[i];
+            }
+        } else {
+            // FNV-1a hash of full name for a unique suffix.
+            uint32_t hash = 2166136261u;
```

Clang (14.0.6)

[warning] variable 'hash' is not initialized

(cppcoreguidelines-init-variables)